tests/jobs/design2params/test_design2params.py

```diff
@@ -178,7 +178,8 @@ def test_support_multiple_formats(tmp_path, ext, engine):
     pd.testing.assert_frame_equal(df, loaded_df, check_dtype=False)
 
 
-def test_run_realization_not_exist(input_data):
+@pytest.mark.usefixtures("input_data")
+def test_run_realization_not_exist():
     with pytest.raises(SystemExit):
```

Comment:
This test is passing even without fixture

Reply:
fixed by asserting the xls file is there.